resources/definitions/os_discovery/enexus.yaml

```diff
@@ -68,6 +73,35 @@ modules:
                         - { descr: critical, graph: 0, value: 18, generic: 2 }
                         - { descr: warning, graph: 0, value: 19, generic: 1 }
                     skip_values: 4
+                -
+                    oid: mainsMainsFailure
```

Comment:
Can you update all references to OIDs to use the MIB name as well like `SP2-MIB::mainsMainsFailure`, not just oid but value, description, etc as well